test_translation_presets.py

```
@@ -0,0 +1,236 @@
+    """Tests für die Anwendung der Translation-Parameter"""
+
+    def setUp(self):
+        """Setup für Parameter-Tests"""
+        self.translator = SubtitleTranslator()
+
+    @patch('translator.SMART_TRANSLATION_AVAILABLE', True)
+    @patch('translator.OPENAI_PROVIDER_AVAILABLE', True)
+    @patch('translator.load_env_vars')
+    @patch('translator.OpenAITranslator')
+    @patch('translator.smart_translate_srt')
+    def test_german_translation_uses_video_optimized_settings(self, mock_translate, mock_provider_class, mock_load_env):
```

Ruff (0.12.2)

Unused method argument: `mock_load_env`

(ARG002)